### PR TITLE
Decommission News Alert(s)

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

Decommission News Alert(s), which has been abandoned as a feature. This is just code cleaning, companion PR of https://github.com/guardian/platform/pull/1447

ps: Also remove a duplicate mention of 
```
(https://github.com/guardian/frontend/blob/main/admin/conf/routes)
```

In documentation

### CODE REVIEW

File: docs/02-architecture/01-applications-architecture.md [hunk 1]
```diff
@@ -120,12 +120,6 @@ Rss app is rendering the RSS version for all Guardian content.
 # Admin
 Admin app hosts a set of dashboards and tools used by Guardian developers to monitor, manage and troubleshoot the Guardian website.
 
-[All supported routes](https://github.com/guardian/frontend/blob/main/admin/conf/routes)
-
-- Create breaking-news alerts
-
-- [Get all breaking-news alert](http://api.nextgen.guardianapps.co.uk/news-alert/alerts)
-
 [All supported routes](https://github.com/guardian/frontend/blob/main/sport/conf/routes)
```
Comment (nicl) [inline docs/02-architecture/01-applications-architecture.md:123]: Wasn't sure why this points to `sport` now (although it seems to predate this change)?

**Review by nicl — APPROVED** [review]
Nice

File: docs/02-architecture/01-applications-architecture.md (hunk 1 above)
Comment (shtukas) [reply docs/02-architecture/01-applications-architecture.md:123]: Oh, Yeah. I removed the duplicate but didn't actually check the link. I suspect the file 01-applications-architecture.md itself might need some cleaning. Will do next.

**Comment by arelra** [comment]
JS counterpart: https://github.com/guardian/frontend/pull/23717

**Pull request merged**